main/GarnetServer/GarnetServer.csproj

```diff
@@ -4,12 +4,10 @@
     <OutputType>Exe</OutputType>
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <PackageId>garnet-server</PackageId>
-    <PackAsTool>true</PackAsTool>
```

Comment:
why is this removed? isnt this what packs the project as a dotnet tool?

Reply:
Yes ... that is true but was trying to confirm something that I was seeing locally.

Reply:
Putting it back to PackAsTool